data-science-pipelines-operator/base/params.env

```diff
@@ -7,3 +7,6 @@ IMAGES_MOVERESULTSIMAGE=registry.redhat.io/ubi8/ubi-micro:8.7
 IMAGES_MARIADB=registry.redhat.io/rhel8/mariadb-103:1
 IMAGES_DSPO=registry.redhat.io/rhods/odh-data-science-pipelines-operator-controller-rhel8:latest
 IMAGES_OAUTHPROXY=registry.redhat.io/openshift4/ose-oauth-proxy:v4.12.0
+IMAGES_MLMDENVOY=quay.io/opendatahub/ds-pipelines-metadata-envoy:v1.0.0
+IMAGES_MLMDGRPC=quay.io/opendatahub/ds-pipelines-metadata-grpc:v1.0.0
+IMAGES_MLMDWRITER=quay.io/opendatahub/ds-pipelines-metadata-writer:v1.0.0
```

Comment:
Please note that this adding a feature that is not enabled by default as it's incubating upstream, but we need to add the default image values here. These will not be leveraged in rhods.